DESCRIPTION

```diff
@@ -21,7 +21,7 @@ Imports:
     methods,
     evaluate (>= 0.5.4),
     IRdisplay,
-    rzmq (>= 0.7.0),
+    pbdZMQ (>= 0.2.1),
```

Comment:
Just to check, this is a requirement on a version that's not yet released on CRAN, right? I'd like to wait for a release and then update the Travis instructions to build it before we merge this.